### PR TITLE
remove local help files for tutorials

### LINKED ISSUE #1626: /download/desktop/burn-a-dvd-on-macos/ to VBT

### LINKED ISSUE #1627: /download/desktop/burn-a-dvd-on-ubuntu/ to VBT

### LINKED ISSUE #1628: /download/desktop/burn-a-dvd-on-windows/ to VBT

### LINKED ISSUE #1630: /download/desktop/create-a-usb-stick-on-macos/ to VBT

### LINKED ISSUE #1631: /download/desktop/create-a-usb-stick-on-ubuntu/ to VBT

### LINKED ISSUE #1632: /download/desktop/create-a-usb-stick-on-windows/ to VBT

### LINKED ISSUE #1633: /download/desktop/install-ubuntu-desktop/ to VBT

### LINKED ISSUE #1635: /download/desktop/try-ubuntu-before-you-install/ to VBT

### LINKED ISSUE #1636: /download/desktop/upgrade to VBT

### LINKED ISSUE #1637: /download/how-to-verify/ to VBT

### LINKED ISSUE #1640: /download/server/install-ubuntu-server/ to VBT

### LINKED ISSUE #1644: /download/server/provisioning to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done 

* updated links to use new tutorials
* removed pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the /download section, including thank-you pages to confirm they are all moved and external links correct

## Issue / Card

Fixes #1640 - /download/server/install-ubuntu-server/ to VBT Status 
Fixes #1637 - /download/how-to-verify/ to VBT Status 
Fixes #1636 - /download/desktop/upgrade to VBT Status 
Fixes #1635 - /download/desktop/try-ubuntu-before-you-install/ to VBT Status 
Fixes #1633 - /download/desktop/install-ubuntu-desktop/ to VBT Status 
Fixes #1632 - /download/desktop/create-a-usb-stick-on-windows/ to VBT Status 
Fixes #1631 - /download/desktop/create-a-usb-stick-on-ubuntu/ to VBT Status 
Fixes #1630 - /download/desktop/create-a-usb-stick-on-macos/ to VBT Status 
Fixes #1628 - /download/desktop/burn-a-dvd-on-windows/ to VBT Status 
Fixes #1627 - /download/desktop/burn-a-dvd-on-ubuntu/ to VBT Status 
Fixes #1626 - /download/desktop/burn-a-dvd-on-macos/ to VBT Status

### CODE REVIEW

File: templates/download/desktop/index.html [hunk 1]
```diff
@@ -89,10 +89,10 @@
             <h3 class="p-heading-icon__title">From an older version</h3>
           </div>
         </div>
-        <p class="p-card__content">If you&rsquo;re already running Ubuntu, you can upgrade in a few clicks from <a href="/download/desktop/upgrade">Software Updater</a>.</p>
+        <p class="p-card__content">If you&rsquo;re already running Ubuntu, you can upgrade in a few clicks from <a href="https://tutorials.ubuntu.com/tutorial/tutorial-upgrading-ubuntu-desktop">Software Updater</a>.</p>
```
Comment (anthonydillon) [inline templates/download/desktop/index.html]: External?

File: templates/download/desktop/index.html [hunk 2]
```diff
@@ -89,10 +89,10 @@
...
         <ul class="p-list">
-          <li class="p-list__item"><a href="/download/desktop/burn-a-dvd-on-ubuntu">How to burn a DVD on Ubuntu&nbsp;&rsaquo;</a></li>
-          <li class="p-list__item"><a href="/download/desktop/create-a-usb-stick-on-ubuntu">How to create a bootable USB stick on Ubuntu&nbsp;&rsaquo;</a></li>
+          <li class="p-list__item"><a href="https://tutorials.ubuntu.com/tutorial/tutorial-burn-a-dvd-on-ubuntu">How to burn a DVD on Ubuntu&nbsp;&rsaquo;</a></li>
+          <li class="p-list__item"><a href="https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-ubuntu">How to create a bootable USB stick on Ubuntu&nbsp;&rsaquo;</a></li>
```
Comment (anthonydillon) [inline templates/download/desktop/index.html]: Externals?

File: templates/download/desktop/index.html [hunk 3]
```diff
@@ -105,8 +105,8 @@ <h3 class="p-heading-icon__title">From Windows</h3>
         </div>
         <p class="p-card__content">If you&rsquo;re using Windows 8 or any computer with a 64-bit processor, we recommend the 64-bit download.</p>
         <ul class="p-list">
-          <li class="p-list__item"><a href="/download/desktop/burn-a-dvd-on-windows">How to burn a DVD on Windows&nbsp;&rsaquo;</a></li>
-          <li class="p-list__item"><a href="/download/desktop/create-a-usb-stick-on-windows">How to create a bootable USB stick on Windows&nbsp;&rsaquo;</a></li>
+          <li class="p-list__item"><a href="https://tutorials.ubuntu.com/tutorial/tutorial-burn-a-dvd-on-windows">How to burn a DVD on Windows&nbsp;&rsaquo;</a></li>
+          <li class="p-list__item"><a href="https://tutorials.ubuntu.com/tutorial/tutorial-create-a-usb-stick-on-windows">How to create a bootable USB stick on Windows&nbsp;&rsaquo;</a></li>
```
Comment (anthonydillon) [inline templates/download/desktop/index.html]: External?

File: templates/download/index.html [hunk 4]
```diff
@@ -33,7 +33,7 @@ <h2 class="p-card__title"><a href="/download/desktop">Ubuntu Desktop&nbsp;&rsaqu
             </div>
           </div>
           <footer class="p-card--ubuntu-upgrade__footer">
-            Do you want to upgrade? <a href="/download/desktop/upgrade">Follow our simple guide&nbsp;&rsaquo;</a>
+            Do you want to upgrade? <a href="https://tutorials.ubuntu.com/tutorial/tutorial-upgrading-ubuntu-desktop">Follow our simple guide&nbsp;&rsaquo;</a>
           </footer>
```
Comment (anthonydillon) [inline templates/download/index.html]: External

**Comment by pmahnke** [comment]
@anthonydillon I updated all these... and I pushed tutorials up yesterday.

**Comment by yaili** [comment]
I'm not sure what I should be reviewing…

**Comment by anthonydillon** [comment]
Code looks good :+1:

**Comment by yaili** [comment]
@pmahnke @anthonydillon can you please tell me what I should be reviewing here?

File: templates/ubuntu-ia.txt [hunk 5]
```diff
@@ -70,26 +70,26 @@ Contributing to Ubuntu TV | Ubuntu TV | Ubuntu	https://www.ubuntu.com/tv/contrib
 TV features and specifications | Ubuntu TV | Ubuntu	https://www.ubuntu.com/tv/features-and-specs/
 Experience Ubuntu TV | Ubuntu TV | Ubuntu	https://www.ubuntu.com/tv/experience/
 Commercial info | Ubuntu TV | Ubuntu	https://www.ubuntu.com/tv/commercial-info/
-How to burn a DVD on macOS | Ubuntu	https://www.ubuntu.com/download/desktop/burn-a-dvd-on-macos/
-Instructions on how to upgrade your Ubuntu desktop. | Ubuntu	https://www.ubuntu.com/download/desktop/upgrade/
+How to burn a DVD on macOS | Ubuntu	https://www.ubuntu.comhttps://tutorials.ubuntu.com/tutorial/tutorial-burn-a-dvd-on-macos/
```
Comment (barrymcgee) [inline templates/ubuntu-ia.txt]: This looks incorrect?

**Comment by grahambancroft** [comment]
Fixing this one up

File: templates/ubuntu-ia.txt (hunk 5 above)
Comment (pmahnke) [inline templates/ubuntu-ia.txt]: fixed

**Comment by pmahnke** [comment]
@grahambancroft. I didn't see you self assigned. I fixed this

**Comment by anthonydillon** [comment]
@pmahnke You have conflicts. Also shouldn't we release the tutorials before this branch lands?

**Comment by anthonydillon** [comment]
@yaili Go to download section and click on any of the installation instruction links. It should take you to a tutorial page.

**Comment by nottrobin** [comment]
Okay I've added those redirects in https://github.com/canonical-websites/www.ubuntu.com/pull/1898/commits/2b7bb241631639e13e97d4e1be1becc0379a6937, and removed the claim to fixing #1644. Now I'm happy.

File: redirects.yaml [hunk 6]
```diff
@@ -776,7 +787,7 @@ employment/?: "http://www.canonical.com/careers"
 employment/SSEBzr/?: "http://www.canonical.com/careers"
 error/?: "/"
 es/?: "/"
-esm/?: "https://insights.ubuntu.com/2017/03/14/introducing-ubuntu-12-04-esm-extended-security-maintenance/"
+esm/?: "/support/eWeek_and_DesktopLinux_declare_Ubuntu_the_Champ"
```
Comment (nottrobin) [inline redirects.yaml]: This doesn't exist...

**Comment by nottrobin** [comment]
All tutorials are now live, I'll merge

**Pull request merged**